core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1258,6 +1258,14 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.MASTER)
           .build();
+  public static final PropertyKey MASTER_EMBEDDED_JOURNAL_SINGLE_MASTER_ELECTION_TIMEOUT =
+      new Builder(Name.MASTER_EMBEDDED_JOURNAL_SINGLE_MASTER_ELECTION_TIMEOUT)
+          .setDescription(
+              "The election timeout to use for single master embedded journal cluster.")
+          .setDefaultValue("500ms")
+          .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
+          .setScope(Scope.MASTER)
+          .build();
```

Comment:
Is there a case when a user would want to configure this? I would like to avoid adding confs if possible.

Reply:
Agreed.